apps/pasaport/.env.example

```diff
@@ -9,3 +9,4 @@ DISCORD_SECRET=''
 TWITCH_ID=''
 TWITCH_SECRET=''
 NEXTAUTH_SECRET='Example'
+GQL_URL="http://localhost:3001/graphql"
```

Comment:
We will need to rename this to `NEXT_PUBLIC_GQL_URL` as this is required on clientside as well. Working on it on #572